db/institutions.yml

```diff
@@ -424,7 +424,8 @@ ubelgrade:
   aka: [University of Belgrade, Универзитет у Београду, Univerzitet u Beogradu]
   city: Belgrade
   country: Serbia
-  institutes: {physics: {name: Institute of Physics, aka: [IPB, Institue of Physics Belgrade]}}
+  departments: {physics: {name: Institute of Physics, aka: [IPB, Institue of Physics Belgrade]}}
   name: University of Belgrade
-  state: ''
+  street: Studentski trg 1
+  state: Beograd
```

Comment:
Please check.  Is Beograd the city (i.e., Belgrade in Serbian)?

Reply:
Ah, you're right, it is. I'll remove the state line.